tools/internal_ci/linux/grpc_bazel_build_in_docker.sh

```diff
@@ -24,4 +24,6 @@ git clone /var/local/jenkins/grpc /var/local/git/grpc
 && git submodule update --init --reference /var/local/jenkins/grpc/${name} \
 ${name}')
 cd /var/local/git/grpc
-bazel build --spawn_strategy=standalone --genrule_strategy=standalone :all test/... examples/...
+# grep lines not starting with INFO to avoid messages when using Bazel wrapper
+bazel build --spawn_strategy=standalone --genrule_strategy=standalone \
+$(bazel query 'kind(cc_.*, :all)' | grep -v "^INFO\: ") test/... examples/...
```

Comment:
looking at this command,
how come the https://github.com/grpc/grpc/blob/809e7c951358a80182d7126b255c3a40881fb3fa/tools/internal_ci/linux/grpc_bazel_on_foundry_base.sh#L39 is not broken by https://github.com/grpc/grpc/pull/19621?

Reply:
I think it's because no objc rules are added to //test yet.